### PR TITLE
working-groups: add dm-portal working group

### PR DESCRIPTION
**Pull request by baurine**
- add dm-portal working group

### CODE REVIEW

File: working-groups/README.md [hunk 1]
```diff
@@ -10,10 +10,11 @@ policies.
...
-* [wg-tidb-sequence](./wg-tidb-sequence.md)
-* [wg-multi-dc-enhancement (TiKV)](https://github.com/tikv/community/tree/master/wg/multiple-dc-enhancemant)
+- [wg-vec-expr](./wg-vec-expr.md)
+- [wg-sql-diagnostics](./wg-sql-diagnostics.md)
+- [wg-key-visualizer](./wg-key-visualizer.md)
+- [wg-wasm](./wg-wasm.md)
+- [wg-dynamic-configuration-change](./wg-dynamic-configuration-change.md)
+- [wg-tidb-sequence](./wg-tidb-sequence.md)
+- [wg-multi-dc-enhancement (TiKV)](https://github.com/tikv/community/tree/master/wg/multiple-dc-enhancemant)
+- [wg-dm-portal](./wg-dm-portal.md)
```
Comment (zz-jason) [inline working-groups/README.md]: missing file `wg-dm-portal.md`?

File: working-groups/wg-dm-portal.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+- [ericsyh](https://github.com/ericsyh), [PingCAP](https://github.com/pingcap)
+
+## Organizers
+
+- [baurine](https://github.com/baurine), [PingCAP](https://github.com/pingcap)
+
+## Members
+
+- [WangXiangUSTC](https://github.com/WangXiangUSTC)
```
Comment (zz-jason) [inline working-groups/wg-dm-portal.md]: please add the organization.

File: working-groups/wg-dm-portal.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# DM Portal Working Group
+
+## Goals
+
+Build a small web app to help generate the DM config file easier.
+
+## Meetings
+
+To be determined.
```
Comment (zz-jason) [inline working-groups/wg-dm-portal.md]: please add the public meeting note and zoom urls.

Comment (baurine) [reply working-groups/wg-dm-portal.md]: ok, I will add it later after it is determined.

File: working-groups/wg-dm-portal.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# DM Portal Working Group
+
+## Goals
+
+Build a small web app to help generate the DM config file easier.
```
Comment (zz-jason) [inline working-groups/wg-dm-portal.md]: do we have any design doc about shit web app?

File: working-groups/wg-dm-portal.md (hunk 3 above)
Comment (baurine) [reply working-groups/wg-dm-portal.md]: @zz-jason already added it, PTAL

File: working-groups/wg-dm-portal.md (hunk 4 above)
Comment (baurine) [reply working-groups/wg-dm-portal.md]: yep, let me find it out and attach the link.

Comment (baurine) [reply working-groups/wg-dm-portal.md]: @zz-jason , added them below, PTAL

File: working-groups/wg-dm-portal.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+- [ericsyh](https://github.com/ericsyh), [PingCAP](https://github.com/pingcap)
+
+## Organizers
+
+- [baurine](https://github.com/baurine), [PingCAP](https://github.com/pingcap)
+
+## Members
+
+- [WangXiangUSTC](https://github.com/WangXiangUSTC), [PingCAP](https://github.com/pingcap)
```
Comment (gingerkidney) [inline working-groups/wg-dm-portal.md]: please add contact for slack channel

Comment (baurine) [reply working-groups/wg-dm-portal.md]: @gingerkidney done, PTAL

**Pull request merged**